### DISCUSSION TITLE
Linkerd inject not working

### DISCUSSION THREAD

**Discussion by storehubdeploy** [linkerd/linkerd2#7355] [General] [answered]
we need grpc load balance, so i installed the linkerd in our k8s cluster.  i use annotation linkerd.io/inject: enabled auto inject, but not workin, no linkerd-init and linkerd-proxy pod attach to the service pod. i use command inject manually is not working too.  can not find any error info either.is any one know how to the loacate the error reason?

Kubenetes cluster version 1.16.15
Linkerd version stable-2.10.2

**Comment by cpretzer** [comment, marked as answer]
hi @storehubdeploy I just added a reply to #7356
